rented_dream/HWstyless/HWContactUs.scss

```diff
@@ -0,0 +1,176 @@
+
+.contact-us {
+    main {
+        display: flex;
+        position: relative;
+        top: 90px;
+
+        .container {
+            width: 100%;
+            max-width: 800px;
+            margin: 0 auto 100px;
+            padding: 0 15px;
+        }
+    }
+    h1 {
+        text-align: center;
+        margin: 5px 0 0 0;
+        color: var(--highlight-color);
+    }
+    h2 {
+        text-align: center;
+        margin-bottom: 5px;
+        color: var(--highlight-color);
+    }
+}
+.contact-info p, .contact-info span {
```

Comment:
these selector also had to be rewrited to SCSS

Reply:
fixed. I used @extend